.travis.yml

```diff
@@ -58,3 +63,21 @@ notifications:
     secure: A19kphrabQHO8TU6qZcBaLQxdSNpm1ypEtbQsh8Ucg6HYPP7y1q7O7JZEndoMRHE9CNKZ9oXQzqR8H1IFVTlnjFFIJfkZzZ1YSNk4abSomhpWCq9daKMfwlcuTtY6PeI1nDVpka4/hiJGn9qzmaKYXle9Sl4CX2VEYp8o8PgMEs=
     rooms:
       secure: FsKzp4ItmOqd/YxqgsElgfjGW2/TU03p2p3ss+PQl/pKDQNnR/2b4pWCQ7GuqYibkmtiH1jYwrnuaLN4Cc+JyN7Z+zUtO4VSORsh3zt/gTsfgphMpCP6cB4sTqUh6AWsZOgzikj+fh7ORHEXVswQwlRHErTgZVEdEkWHBh4UWzc=
+
+before_install:
+- test $TRAVIS_BRANCH != coverity_scan -o ${TRAVIS_JOB_NUMBER##*.} = 1 || exit 0
+- echo -n | openssl s_client -connect https://scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
+
+env:
+  global:
+    secure: cSKI7GYTFP8VQZYMT6Fo0XQjx3r+yXf0b1r3y8NNWXOXg8xIAXFE3lM5+kvv/eDPz43sknskwmb0e093WzKN6Ht9ngJ1nsUh/cleFwPjGgYiYF4/EfMR5MhNrWQyYSbT0g2gF9+jJG1mOmXZ3cvMVzkuuJluOPB1E8tZX9T+ml8=
+
+addons:
+  coverity_scan:
+    project:
+      name: racket/racket
+      description: The Racket Programming Language
+    notification_email: pmatos@linki.tools
```

Comment:
Your personal e-mail?

Reply:
Thanks. I forgot to change that. However, now that I think about it, actually my email is probably correct as I would be heading this effort as discussed with @samth . Maybe @samth  wants to get Coverity results as well?